DNN Platform/DotNetNuke.Web.Mvc/Extensions/StartupExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DotNetNuke.Web.Mvc.Framework.Controllers;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Linq;
+
+namespace DotNetNuke.Web.Mvc.Extensions
+{
+    internal static class StartupExtensions
+    {
+        public static void AddMvc(this IServiceCollection services)
```

Comment:
If we can get some XML comments to document purpose here as well

Reply:
Added xml comments